packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -109,14 +110,22 @@ function SidePanel({
 	const toggleButtonTitle = docked ? expandLabel : collapseTitle;
 	const Components = Inject.getAll(getComponent, { Action, ActionList });
 	return (
-		<nav
-			id={id}
-			className={navCSS}
-			role="navigation"
-			aria-expanded={!((dockable && docked) || minimised)}
-			ref={ref}
-			style={{ width }}
-		>
+		<nav id={id} className={navCSS} role="navigation" ref={ref} style={{ width }}>
+			{backgroundIcon && (
+				<style>
+					{`#${id}::before {
+						content: '';
+						position: absolute;
+						left: 0;
+						bottom: -50px;
+						height: 31rem;
+						width: 31rem;
+						background-repeat: no-repeat;
+						background-color: rgba(255, 255, 255, 0.1);
+						mask-image: url('${backgroundIcon}');
```

Comment:
Seems like it wont generate a -webkit-mask-image attribute for chrome when in a <style> element , can you confirm ?